packages/gestalt/src/Text.js

```diff
@@ -62,23 +70,7 @@ export default function Text({
   const cs = cx(
     styles.Text,
     styles[`fontSize${scale}`],
-    color === 'blue' && colors.blue,
-    color === 'darkGray' && colors.darkGray,
-    color === 'eggplant' && colors.eggplant,
-    color === 'gray' && colors.gray,
-    color === 'green' && colors.green,
-    color === 'lightGray' && colors.lightGray,
-    color === 'maroon' && colors.maroon,
-    color === 'midnight' && colors.midnight,
-    color === 'navy' && colors.navy,
-    color === 'olive' && colors.olive,
-    color === 'orange' && colors.orange,
-    color === 'orchid' && colors.orchid,
-    color === 'pine' && colors.pine,
-    color === 'purple' && colors.purple,
-    color === 'red' && colors.red,
-    color === 'watermelon' && colors.watermelon,
-    color === 'white' && colors.white,
+    color && colors[color],
```

Comment:
This has no change in the output, just a refactor to simplify

Reply:
@dangerismycat it looks like this does include a change in behavior. E.g. we can now use `transparentDarkGray` as a text color which is probably not desired.

Might still be good to have an includes list: `color && ['white', ...].includes(color) && colors[color]`

Reply:
Good catch! Updating